### ISSUE TITLE
Dropbox link of RetinaFace Anti Cov Face Detector pre-trained is unavailable

### ISSUE THREAD

**Issue by tungedng2710** [deepinsight/insightface#1788]
I want to get MobileNet pretrained of RetinaFace model for face mask detection, but your Dropbox link seemed to be unavailable and I have Baidu link only. I couldn't sign up and get it from Baidu, are there any alternative ways?

**Comment by fuenwang** [comment]
Hi @nttstar,

Could you provide me retinaface-R50 pretrained model? The dropbox link is dead too.

Best regards,
Fu-En Wang

**Comment by LeeKyungwook** [comment]
Any update??
